### DISCUSSION TITLE
alignment5

### DISCUSSION THREAD

**Discussion by giscus** [jamesstephenbrown/NZSWebsite#24] [Announcements]
# alignment5

Exploring the alignment of individual and collective interests in the realm of politics and society.

https://nonzerosum.games/alignment5.html

**Comment by andrewtaneglen** [comment]
This would be made easier when something like chat-gpt is aware of the context of your individual life, and can provide alignment insight in a deeply contextual way, like a friend you had your whole life. Either that or we do a version of that black mirror episode where they augment your vision to make other people look like monster/aliens, except it makes everyone else look like a member of your family, or someone else you care for.

**Reply by jamesstephenbrown** [reply]
I think that could actually make a very good Black Mirror episode, exploring what happens when you lose any xenophobia at all, sounds like something that could easily exploited.
